C-Plus-Plus/ds/max_min_heap.cpp

```diff
@@ -0,0 +1,118 @@
+#include<bits/stdc++.h>
+using namespace std;
+class heap{
+    int n;
+    int arr[1000];
```

Comment:
Don't take fixed values. Define the array after declaring n, like **a[n]**
Or better use vector for the same